client/state/global-sidebar/selectors.ts

```diff
@@ -23,6 +23,10 @@ const SITE_DASHBOARD_ROUTES = {
 	'site-marketing': '/sites/marketing',
 	'site-tools': '/sites/tools',
 	'site-settings': '/sites/settings',
+
+	// Domain Management
+	'all-domain-management': '/domains/all/manage',
+	'all-email-management': '/domains/all/email',
```

Comment:
The email-related one is for the future, I suppose? (Didn't find other mentions of it in this PR.)

Reply:
Yep it is for the emails tab, when we add something in it.